### PR TITLE
fix shellcheck in k8s.io/apiserver

### PR DESCRIPTION
**Pull request by danielqsj**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

fix shellcheck failures in `staging/src/k8s.io/apiserver`

**Which issue(s) this PR fixes**:

work towards #72956

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/gencerts.sh [hunk 1]
```diff
@@ -83,20 +83,20 @@ See the License for the specific language governing permissions and
...
+// This file was generated using openssl by the gencerts.sh script
+// and holds raw certificates for the webhook tests.
+
+package webhook
 EOF
 
-echo "// This file was generated using openssl by the gencerts.sh script" >> $outfile
-echo "// and holds raw certificates for the webhook tests." >> $outfile
-echo "" >> $outfile
-echo "package webhook" >> $outfile
```
Comment (BenTheElder) [inline staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/gencerts.sh]: huh I wonder why we were doing this before ... 🤷‍♂️

**Review by BenTheElder — APPROVED** [review]
/lgtm
thanks

**Review by fejta — COMMENTED** [review]
/approve
for hack

**Comment by danielqsj** [comment]
@lavalamp @smarterclayton @deads2k @sttts @liggitt if you have time, can you help review this? Thanks

File: staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/testcerts/gencerts.sh [hunk 2]
```diff
@@ -90,20 +90,20 @@ See the License for the specific language governing permissions and
...
 	echo "" >> $outfile
 	echo "var $file = []byte(\`$data\`)" >> $outfile
 done
 
 # Clean up after we're done.
-rm *.pem
-rm *.csr
-rm *.srl
-rm *.conf
+rm ./*.pem
```
Comment (sttts) [inline staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/testcerts/gencerts.sh]: wow, really? This is necessary?

Comment (danielqsj) [reply staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/testcerts/gencerts.sh]: `shellcheck` think it as warning. :(

**Review by logicalhan — APPROVED** [review]
/lgtm

**Pull request merged**